curriculum/challenges/english/25-front-end-development/quiz-css-colors/66ed8fe1f45ce3ece4053eb1.md

```diff
@@ -17,439 +17,487 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+Which of the following is a valid way to represent a color in CSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+`color-value: black;`
 
 ---
 
-Placeholder distractor 2
+`text-color: blue;`
 
 ---
 
-Placeholder distractor 3
+`font-color: red;`
 
 #### --answer--
 
-Placeholder answer
+`#ff5733`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the correct syntax for setting the background color of an element?
 
 #### --distractors--
 
-Placeholder distractor 1
+`bg-color: red;`
 
 ---
 
-Placeholder distractor 2
+`background_color: red;`
 
 ---
 
-Placeholder distractor 3
+`style: background-color: red;`
 
 #### --answer--
 
-Placeholder answer
+`background: red;`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How can you set the transparency of a color to 50%?
```

Comment:
```suggestion
How can you set the transparency of an element to 50%?
```

Colour transparency is managed by the alpha channel, the `opacity` property affects the entire element(s) targeted by the selector.